attestation-agent/attestation-agent/src/bin/grpc-aa/main.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+mod server;
+
+use anyhow::*;
+use clap::Parser;
+use log::debug;
+
+use std::net::SocketAddr;
+
+const DEFAULT_ATTESTATION_AGENT_ADDR: &str = "127.0.0.1:50002";
+
+#[derive(Debug, Parser)]
+#[command(author, version, about, long_about = None)]
+struct Cli {
+    /// Attestation gRPC Unix socket addr.
+    ///
+    /// This Unix socket address which the Attestation ttRPC service
+    /// will listen to, for example:
+    ///
+    /// `--attestation_sock 127.0.0.1:11223`
+    #[arg(default_value_t = DEFAULT_ATTESTATION_AGENT_ADDR.to_string(), short, long = "attestation_sock")]
+    attestation_sock: String,
+}
+
+#[tokio::main]
+pub async fn main() -> Result<()> {
```

Comment:
We don't need SIGINT, etc handlers for grpc main? (like we do on ttRPC impl)

Reply:
Nice suggestion. Let me add that.